File: JSONPlaces/UITableViewController+gesture.m
//
//  UITableViewController+gesture.m
//  JSONPlaces
//
//  Created by Adrian Kubała on 12.02.2017.
//  Copyright © 2017 Adrian. All rights reserved.
//

#import "UITableViewController+gesture.h"

@implementation UITableViewController (gesture)

-(void)addLongPressGestureRecognizer:(SEL)selector {
  UILongPressGestureRecognizer *recognizer = [[UILongPressGestureRecognizer alloc] initWithTarget:self action:selector];
  [self.tableView addGestureRecognizer:recognizer];
}

@end

File: JSONPlaces/JSONPlaces/UITableViewController+gesture.swift
//
//  UITableViewController+gesture.swift
//  JSONPlaces
//
//  Created by Adrian Kubała on 09.02.2017.
//  Copyright © 2017 Adrian. All rights reserved.
//

import UIKit

extension UITableViewController {
  func addLongPressGestureRecognizer(selector: Selector) {
    let recognizer = UILongPressGestureRecognizer.init(target: self, action: selector)
    tableView.addGestureRecognizer(recognizer)
  }
}
